Extensions/outbox.js

```diff
@@ -177,7 +177,9 @@ XKit.extensions.outbox = new Object({
 
 		var form_key = XKit.interface.form_key();
 
-		$(document).on("click", "#ask_button", function() {
+		$(document).on("click", ".post-form--asks .ask-button", function() {
+
+			var $form = $(this).closest('form');
```

Comment:
I'd rather write something that is more flexible to tumblr changes. But I do agree the selector could be more specific to make sure it will always return either the correct element or nothing.